pipeline/beam_tables.py

```diff
@@ -542,7 +553,8 @@ def run_beam_pipeline(self, scan_type: str, incremental_load: bool,
                         gcs_folder: Optional[str],
                         start_date: Optional[datetime.date],
                         end_date: Optional[datetime.date],
-                        export_gcs: bool) -> None:
+                        export_gcs: bool,
+                        bq_and_gcs: bool) -> None:
```

Comment:
I think out would be best to have 

export_gcs and export_bq 

as separate bool options, and then having both false raise an error.

Reply:
Done